docs/reference/migration/migrate_8_0/node.asciidoc

```diff
@@ -46,103 +46,6 @@ all the corresponding subfolders in parallel. Each node uses the same subfolder
 (e.g. `nodes/2`) across all its data paths.
 ====
 
-.Support for multiple data paths has been removed.
-[%collapsible]
-====
-*Details* +
-In earlier versions the `path.data` setting accepted a list of data paths, but
-if you specified multiple paths then the behaviour was unintuitive and usually
-did not give the desired outcomes. Support for multiple data paths is now
-removed.
-
-*Impact* +
-Specify a single path in `path.data`. If needed, you can create a filesystem
-which spans multiple disks with a hardware virtualisation layer such as RAID,
-or a software virtualisation layer such as Logical Volume Manager (LVM) on
-Linux or Storage Spaces on Windows. If you wish to use multiple data paths on a
-single machine then you must run one node for each data path.
-
-If you currently use multiple data paths in a
-{ref}/high-availability-cluster-design.html[highly available cluster] then you 
-can migrate to a setup that uses a single path for each node without downtime 
-using a process similar to a 
-{ref}/restart-cluster.html#restart-cluster-rolling[rolling restart]: shut each
-node down in turn and replace it with one or more nodes each configured to use
-a single data path. In more detail, for each node that currently has multiple
-data paths you should follow the following process. In principle you can
-perform this migration during a rolling upgrade to 8.0, but we recommend
-migrating to a single-data-path setup before starting to upgrade.
-
-1. Take a snapshot to protect your data in case of disaster.
-
-2. Optionally, migrate the data away from the target node by using an
-{ref}/modules-cluster.html#cluster-shard-allocation-filtering[allocation filter]:
-+
-[source,console]
---------------------------------------------------
-PUT _cluster/settings
-{
-  "transient": {
-    "cluster.routing.allocation.exclude._name": "target-node-name"
-  }
-}
---------------------------------------------------
-+
-You can use the {ref}/cat-allocation.html[cat allocation API] to track progress 
```

Comment:
I don't think we want to lose most of this documentation, as it was specifically worked out to help with migration, which users still need to do. Can it be moved somewhere appropriate for that purpose?

Reply:
We'll still have it in the 7.x docs here: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/migrating-7.13.html#breaking_713_infra_core_deprecations

Those docs are also included in any 7.x branch after 7.13.

I can relocate this to a deprecations section in the 8.0 docs as well. That's not standard, but we can make an exception if wanted.

Reply:
Thanks for the discussion @rjernst. I relocated this content over to the path settings page with df2975448e51a33261043a47cd646ab31b3d9e01.